packages/radix/src/components/Code.tsx

```diff
@@ -13,37 +12,18 @@ export const Code = React.forwardRef<HTMLElement, CodeProps>((props, forwardedRe
       base: {
         code: {
           normal: {
+            display: 'inline-block',
```

Comment:
@peduarte curious as to why? Chunkier height? Should it go on the primitive ideally?
cc @benoitgrelard

Reply:
I wouldn't think it'd need it. What happens if you remove it? I think it should be inline to be honest, this component is used in the middle of paragraphs right?

Reply:
I generally agree it should be `inline` as well, but based on the current styles, I interpreted the intent that it needs to have a computed height of `20px`. Since `line-height` property sets the height of a line box, inline-level elements it doesn't impact its computed height

Changing it to `inline-block` makes it so the height becomes `20px`

Although it does introduces other issues such as inability to wrap text on the same line 🙄 

If we go with a more dynamic styling, as Vlad suggested [here](https://github.com/modulz/radix/pull/260#pullrequestreview-408381445) and I suggested many times in the past, then I think we can remove `inline-block` since it no longer seems that we want to set the height base don line height

Reply:
> Changing it to `inline-block` makes it so the height becomes `20px`

Yeah, we can work with the paddings to make it higher then

Reply:
I updated the paddings in https://github.com/modulz/radix/issues/261 to compensate for the inline height